typings/index.d.ts

```diff
@@ -495,6 +499,10 @@ export abstract class Channel extends Base {
 
 export type If<T extends boolean, A, B = null> = T extends true ? A : T extends false ? B : A | B;
 
+export interface ClientFetchInviteOptions {
```

Comment:
Shouldn’t this be at the very bottom in the mixins region?

Reply:
yes, but lazy me just left it there 😴